src/main/docs/guide/appendix/breaks.adoc

```diff
@@ -379,6 +379,10 @@ public class AProvider implements Provider<A> {
 }
 ----
 
+==== `ApplicationEventPublisher` has a generic event type and it's no longer possible to register an event listener after context has started
```

Comment:
The act of registering a listener post context startup is pretty unlikely so imo not worth mentioning here. The improvements. More comprehensive documentation about way it was done before and how it should be done now should be reflected in the whatsNew and upgrading docs.

Reply:
Added whatsNew entry